src/modules/household/components/ManageHousehold.tsx

```diff
@@ -90,24 +89,25 @@ const ManageHousehold = ({
   useScrollToHash(loading || recentMembersLoading);
   const isMobile = useIsMobile();
 
-  const columns: ColumnDef<ClientFieldsFragment | RecentHouseholdMember>[] =
-    useMemo(() => {
-      const defaults = [...householdMemberColumns];
-      if (isMobile) {
-        // On mobile, show enrollment button right next to the client name so user
-        // doesn't have to scroll to the right.
-        defaults.splice(1, 0, ...addToEnrollmentColumns);
-      } else {
-        defaults.push(...addToEnrollmentColumns);
-      }
-      if (globalFeatureFlags?.mciId) {
-        return [
-          externalIdColumn(ExternalIdentifierType.MciId, 'MCI ID'),
-          ...defaults,
-        ];
-      }
-      return defaults;
-    }, [addToEnrollmentColumns, globalFeatureFlags?.mciId, isMobile]);
+  const columns = useMemo(() => {
+    const defaults: ColumnDef<
+      ClientSearchResultFieldsFragment | RecentHouseholdMember
+    >[] = [CLIENT_COLUMNS.name, CLIENT_COLUMNS.dobAge];
+    if (isMobile) {
+      // On mobile, show enrollment button right next to the client name so user
```

Comment:
(no need to make a change in this PR, just mulling) This was an interesting one-off fix, but now that we have sticky columns implemented as a more general pattern, that would be appropriate to apply here? We can check in with design about it, since there is a risk with sticky columns for the total sticky width to get wide enough that the non-sticky columns aren't scrollable at all.

Reply:
Good question, I was wondering the same. We could add `sticky: 'right',` to `addToEnrollmentColumns`:
<img width="421" alt="Screenshot 2025-02-06 at 12 58 34 PM" src="https://github.com/user-attachments/assets/e2e76038-5f17-4d8d-8260-76412fb55525" />

Though in the previous designs we wanted to be able to have that "action button" inside the same cell as the menu (re Bed Nights), and I think we descoped it.